packages/flame/svg/Icons/icon-arrow-down-right.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><g id="arrow-down-right"><path id="base-1" d="M13.533 6.056l.45 7.663a.25.25 0 0 1-.264.264l-7.663-.45a.25.25 0 0 1-.162-.427l1.752-1.752a.5.5 0 0 0 0-.708L2.707 5.707a1 1 0 0 1 0-1.414l1.586-1.586a1 1 0 0 1 1.414 0l4.94 4.94a.5.5 0 0 0 .707 0l1.752-1.753a.25.25 0 0 1 .427.162z"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><g id="small-arrow-down-right" transform="translate(-2 -2)"><g id="arrow-down-right" transform="translate(-1 -1)"><path id="base-1" d="M9.854 2.347L13.7 6.229a.999.999 0 0 1-.078 1.496 1.228 1.228 0 0 1-1.63-.072L9.973 5.614v9.413a1 1 0 1 1-2 0V5.669L6.008 7.653a1.228 1.228 0 0 1-1.63.072.999.999 0 0 1-.078-1.496l3.846-3.882a1.225 1.225 0 0 1 1.708 0z" transform="rotate(135 9 9.013)"/></g></g></svg>
```

Comment:
Did you mean to change `icon-small-arrow-down-right.svg` instead of this one? Would've never seen it if Percy wasn't there 😉 See https://percy.io/Lightspeed/flame/builds/5238858/view/308058043?browser=chrome&mode=diff&width=1280v.